packages/xo-server/src/api/vm.js

```diff
@@ -552,7 +566,7 @@ export const set = defer(async function($defer, params) {
       throw unauthorized()
     }
 
-    await this.setVmResourceSet(vmId, resourceSetId)
+    await this.setVmResourceSet(vmId, resourceSetId, true)
```

Comment:
I agree but this is not directly related to this PR and the critical thing here is making sure that we're not preventing an admin from doing anything, no matter what.

Reply:
I think it's related because this PR change the behavior, without it, the `set` call will fails if the quota exceeds.

If we don't inform the admin that he will exceed the quota, he will never know it.